tools/socketgen/socketgen.py

```diff
@@ -1443,14 +1489,73 @@ def gen_tech_indep_impl(accelerator_list, cache_list, dma_width, template_dir, o
           f.write("end entity " + acc.name + "_rtl;\n\n")
           f.write("\n")
           f.write("architecture mapping of " + acc.name + "_rtl is\n\n")
+# begin insert
```

Comment:
fix indentation or delete these comments...not sure what they mean

Reply:
fixed